### PR TITLE
Add test case for RunCreateToken

### PR DESCRIPTION
**Pull request by xiangpengzhao**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
ref: https://github.com/kubernetes/kubernetes/pull/55377#discussion_r149910681

**Special notes for your reviewer**:
/cc @luxas
**Release note**:

```release-note
none
```

### CODE REVIEW

**Review by luxas — COMMENTED** [review]
/assign @kad

File: cmd/kubeadm/app/cmd/token.go
Comment (xiangpengzhao) [inline cmd/kubeadm/app/cmd/token.go]: This libe was removed in #55377 by mistake. Added here again and it will be removed by #53929.

File: cmd/kubeadm/app/cmd/token_test.go
Comment (xiangpengzhao) [inline cmd/kubeadm/app/cmd/token_test.go]: This should be added after #53929 merged.

Comment (kad) [reply cmd/kubeadm/app/cmd/token_test.go]: I would suggest to wait until #53929 merged, get here negative usages and then merge this PR. Otherwise looks ok for me.

Comment (xiangpengzhao) [reply cmd/kubeadm/app/cmd/token_test.go]: Agreed :)

**Comment by xiangpengzhao** [comment]
It's not worthy being in 1.9. Will leave it alone until I have thread.

**Comment by xiangpengzhao** [comment]
/test pull-kubernetes-unit
`k8s.io/kubernetes/test/integration/garbagecollector TestCRDDeletionCascading 1m22s` :snowflake:

File: cmd/kubeadm/app/cmd/token_test.go
Comment (xiangpengzhao) [reply cmd/kubeadm/app/cmd/token_test.go]: @kad updated.

**Comment by xiangpengzhao** [comment]
/retest

active this thread.

**Review by luxas — APPROVED** [review]
/approve no-issue
Thanks @kad for the review!

**Pull request merged**
